docs/composability/call_solana_interface.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'ICallSolana Interface'
+proofedDate: 20240627
+iterationBy: na
+includedInSite: false
+approvedBy: na
+---
+
+## Introduction
+
+To use the **Composability** feature of Solana, let us learn about the interface and the functions included in the interface that will be used to interact with the Solana programs via the precompile `0xFF00000000000000000000000000000000000006`.
```

Comment:
Since Neon is already on Solana, may be we can say "To leverage the Composability feature of Solana thorugh Neon EVM".
Does this sound okay to you @OlehSyzonov ?

Reply:
Thank you for the answer. I see, it is about Solana. No need to change then.